docs/community/speedup.rst

```diff
@@ -27,17 +27,15 @@ Following those instructions (provided you have the necessary toolchain installe
 
     export CHARSET_NORMALIZER_USE_MYPYC=1
     pip install mypy build wheel
-    pip install charset-normalizer --no-binary :all:
+    pip install charset-normalizer --no-binary charset-normalizer
```

Comment:
That one is a bit repetitive. How about keeping that one as is and adding an example where charset-normalizer is the only one with bin wheel available (in deps tree)?

```
pip install charset-normalizer --no-binary :all:
```

```
pip install requests --no-binary charset-normalizer
```